### ISSUE TITLE
Fix failing UIA tests from #10886

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/terminal#10924]
#10886 follow-up

There's some failing tests in the CSV that we're skipping right now. They shouldn't be failing, but the tests were verified to be correct when comparing our behavior to MS Word's.

We should update our UIA implementation to make these tests not fail.

IIRC I believe the failing tests are mainly:
- "move by line" backwards on degenerate ranges are off by one
- "move by document" do not report that they moved by the correct amount
- a non-degenerate range that moves 0 should still expand by whatever unit you're moving by (even if you tried to move by 5/1/0 but ended up moving 0)

**Comment by carlos-zamora** [comment]
Added to the v1.11 milestone, but feel free to move it elsewhere.

Also, this is in the shared UIA implementation, so the bug affects Windows Terminal and Conhost
